src/devices/IS31FL3730/README.md

```diff
@@ -0,0 +1,37 @@
+﻿# IS31FL3730 - I2C LED Matrix Controller
+The IS31FL3730 is a flexible I2C-based LED Matrix controller.  Each IS31FL3730 can control a pair of LED matrices in
```

Comment:
all titles in Markdown should have on extra enter after them

Reply:
Also please do not use I2C in the device title unless device is providing I2C interface (i.e. Ft4222)